.github/workflows/test.yml

```diff
@@ -0,0 +1,102 @@
+name: Build & Test
```

Comment:
name is `Build & Test` but file is called `test.yml`, I'd rename the file though